packages/mcp/mcp-client-chat/src/ai/ai-sdk/ai-sdk.ts

```
@@ -0,0 +1,205 @@
+  }
+
+  async chatStream(chatBody: ChatBody): Promise<globalThis.ReadableStream<Uint8Array>> {
+    try {
+      const chatOptions: StreamTextOptions = this.generateChatOptions(chatBody);
+      const result: StreamTextResult<ToolSet, unknown> = streamText({
+        model: this.model,
+        ...chatOptions,
+      });
+      const iterator = this.openaiChunkGenerator(result);
+
+      return new ReadableStream<Uint8Array>({
```

ESLint

[error] The 'ReadableStream' is still an experimental feature The configured version range is '^20.19.0 || >=22.12.0'.

(n/no-unsupported-features/node-builtins)